src/common/utils/time-utils.ts

```diff
@@ -69,3 +72,15 @@ function getTimeUnitOption(amount: number, unit: TimeUnit): DurationObject {
       return { months: amount };
   }
 }
+
+export function getStartOf(date: Date, unit: TimeUnit): Date {
+  return moment(date).startOf(unit).toDate();
+  // TODO (Fai): Migrate to below implementation.
+  // return DateTime.fromJSDate(date).startOf(unit).toJSDate();
+}
+
+export function getTimeDiff(date1: Date, date2: Date, unit: TimeUnit) {
+  return moment(date1).diff(date2, unit);
```

Comment:
Consider adding a comment to say what the function does, and maybe an example. I think it's easy to not remember which date are you supposed to pass first, including maybe an example?

```ts
/**
 * Calculate the difference between 2 dates in the given 
 * time unit. Example:
 * 
 *   getTimeDiff(dateTomorrow, dateToday, TimeUnit.DAYS));  // 1
 */
```